requirements.txt

```diff
@@ -1,5 +1,6 @@
 ansible>=2.2
-six
+click
 pyOpenSSL
-PyYAML
-ruamel.yaml
+# We need to disable ruamel.yaml for now because of test failures
```

Comment:
Hmm, the test fix didn't merge yet, right?
I got the error today when my venv had ruamel.yaml installed.